src/V3Fork.cpp

```diff
@@ -34,12 +46,351 @@
 #include "V3Ast.h"
 #include "V3AstNodeExpr.h"
 #include "V3Global.h"
+#include "V3MemberMap.h"
 
 #include <algorithm>
+#include <map>
 #include <set>
+#include <vector>
 
 VL_DEFINE_DEBUG_FUNCTIONS;
 
+class DynScopeInstance final {
+public:
+    AstClass* classp = nullptr;  // Class for holding variables of dynamic scope
+    AstClassRefDType* refDTypep = nullptr;  // RefDType for the above
+    AstVar* handlep = nullptr;  // Class handle for holding variables of dynamic scope
```

Comment:
I thought that only applied to private members and I wanted to avoid creating unnecessary accessors, especially given that I wanted to keep `ForkDynScopeInstance` a _POD_.

I changed the names to include the `m_` prefix though.